src/test/string-utils.test.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert');
+
+const StringUtils = require('../main/lib/string-utils');
+
+describe('StringUtils', () => {
+  describe('splitBy', () => {
+    it('splits by default delimeter and trims the input', () => {
+      const input = ' text , to be ,   splitted   ';
+
+      const result = StringUtils.split(input);
+
+      console.log(result);
```

Comment:
Do we need console log here? Please remove.

Reply:
Done